src/game/Tactical/Soldier_Control.h

```diff
@@ -248,6 +248,8 @@ enum InvSlotPos
 	NUM_INV_SLOTS,
 };
 
+extern UINT8 gubItemDroppableFlag[NUM_INV_SLOTS];
```

Comment:
This invalidates the comment above this array's definition in Soldier_Create.cc. And why did you place this declaration in this header? Seems a bit odd here.

Reply:
Since `gubItemDroppableFlag[]` defines which inv slots can be marked undroppable I placed it under the InvSlotPos enum and at the same time from here it makes it visible to MercProfile.cc.